### PR TITLE
Reduce call on non-null type from !isNullOrEmpty to isNotEmpty

### PR DESCRIPTION
**Pull request by ataulm**
Fixes https://github.com/nickbutcher/plaid/issues/497

Thanks @XinyueZ

### CODE REVIEW

File: core/src/main/java/io/plaidapp/core/dribbble/data/search/DribbbleSearchConverter.kt [hunk 1]
```diff
@@ -63,7 +63,7 @@ object DribbbleSearchConverter : Converter<ResponseBody, List<Shot>> {
         val title = descriptionBlock.select("strong").first().text()
         // API responses wrap description in a <p> tag. Do the same for consistent display.
         var description = descriptionBlock.select("span.comment").text().trim { it <= ' ' }
-        if (!description.isNullOrEmpty()) {
+        if (!description.isEmpty()) {
```
Comment (XinyueZ) [inline core/src/main/java/io/plaidapp/core/dribbble/data/search/DribbbleSearchConverter.kt]: `isNotEmpty` is better, u know what I mean :)

Comment (ataulm) [reply core/src/main/java/io/plaidapp/core/dribbble/data/search/DribbbleSearchConverter.kt]: Ha, it is! Will change it when I'm home thanks 😋

Comment (ataulm) [reply core/src/main/java/io/plaidapp/core/dribbble/data/search/DribbbleSearchConverter.kt]: commit 75bc908e4d1feb9dfa5c96eadcfbd34a4bd7338d (HEAD -> am/description-optional, ataulm/am/description-optional)
Author: Ataul Munim <hello@ataulm.com>
Date:   Sat Feb 2 22:01:25 2019 +0000

    Replace operator negation with built-in function

**Comment by florina-muntenescu** [comment]
@ataulm Thanks for your PR!
@XinyueZ Please let the owners of the repo approve changes. Having non-owners approving changes can lead to confusion for PR requester.

**Pull request merged**